kedro/framework/cli/micropkg.py

```diff
@@ -227,9 +226,11 @@ def _pull_package(  # noqa: PLR0913
         # However, the rest of the code expects the non-normalized package name,
         # so we have to find it.
         packages = [
-            package
-            for package in FlatLayoutPackageFinder().find(project_root_dir)
-            if "." not in package
+            elem.name
```

Comment:
`elem` isn't a very descriptive variable name, can you rename it to be clearer? Something like project_item maybe?

Reply:
Thanks, fixed!